src/OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -117,8 +121,20 @@ static const std::string kLogFileNameDelimiter = "-";
   }
 
 extern std::ofstream log_file;
-std::optional<int> GetLogFileLifetimeMinutes(std::string log_file_name);
-ErrorMessageOr<void> TryRemoveOldLogFiles(const std::filesystem::path& log_dir);
+
+namespace orbit_base {
+[[nodiscard]] uint32_t GetCurrentProcessId();
+[[nodiscard]] std::filesystem::path CreateOrGetLogDir();
+[[nodiscard]] std::filesystem::path GetLogFilePath(const absl::Time& timestamp = absl::Now(),
+                                                   const uint32_t& pid = GetCurrentProcessId());
+}  // namespace orbit_base
+
+[[nodiscard]] std::vector<std::filesystem::path> ListFiles(const std::filesystem::path& log_dir);
+ErrorMessageOr<absl::Duration> GetLogFileLifetime(const std::string& log_file_name);
+[[nodiscard]] std::vector<std::filesystem::path> FilterOldLogFiles(
+    const std::vector<std::filesystem::path>& log_file_paths);
+ErrorMessageOr<void> RemoveFiles(const std::vector<std::filesystem::path>& log_file_paths);
```

Comment:
Maybe a comment here, that this function will remove files even when an error is returned. 

As a reader, I might expect this function to not have delete any files when an error is returned

Reply:
Added a comments here. Thanks for the suggestion.

Reply:
nit: The same comment also for `RemoveFiles` (similar issue)